deployment/commons/sso/keycloak/entrypoint/run-keycloak

```diff
@@ -40,15 +42,11 @@ if [ ! -f /opt/jboss/keycloak/standalone/data/first-run ]; then
   #
   # Keycloak Admin credential configuration
   echo "Configuring Keycloak admin credentials..."
-  $KC config credentials \
-        --server http://localhost:8080/auth \
-        --realm master \
-        --user "$KEYCLOAK_USER" \
-        --password "$KEYCLOAK_PASSWORD" ||
-  {
```

Comment:
The following bit of code used to inform users when admin configuration went wrong, are we sure we want to omit this?